### PR TITLE
[General] Aman'Thul's Vision

### PR DESCRIPTION
**Pull request by CollCrom**
Added in a module for Amanthuls. Also removed friendly fire in all Retribution modules that were accounting for it.

capture

### CODE REVIEW

File: src/common/SPELLS/OTHERS.js [hunk 1]
```diff
@@ -778,4 +778,15 @@ export default {
...
   },
+  //Aman'Thul's Vision
+  AMANTHULS_GRANDEUR: { //Pantheon Proc
+    id: 256832,
+    name: 'Aman\'Thul\'s Grandeur',
+    icon: 'ability_shaman_repulsiontotem',
+  },
+  GLIMPSE_OF_ENLIGHTENMENT: { //Regular Proc
+    id: 256818,
+    name: 'Glimpse of Enlightenment',
```
Comment (Pewtro) [inline src/common/SPELLS/OTHERS.js]: These two `` //Regular Proc `` and ``//Pantheon Proc`` should be opposite right?

Comment (CollCrom) [reply src/common/SPELLS/OTHERS.js]: http://www.wowhead.com/spell=256818/glimpse-of-enlightenment

http://www.wowhead.com/spell=256832/amanthuls-grandeur

I don't think so.

Comment (CollCrom) [reply src/common/SPELLS/OTHERS.js]: done and done friend

**Comment by Pewtro** [comment]
Oi travis speed up

**Pull request merged**